contracts/test/utils/RLPEncodedTransactions.sol

```diff
@@ -38,25 +38,24 @@ contract RLPEncodedTransactionsUtil is Test {
         )
     ];
 
-    function _helper_sequencerInbox_appendTx_Version() internal returns (uint256) {
+    function _helper_sequencerInbox_appendTx_Version() internal pure returns (uint256) {
         // Update this as necessary
         return 0;
     }
 
-    function _helper_sequencerInbox_appendTx(uint256 numTxns) internal view returns (bytes memory, uint256[] memory) {
-        uint256[] memory transactionLengths = new uint256[](numTxns);
+    // TODO: RLP encode txBatch format
+    function _helper_sequencerInbox_appendTx(uint256 numTxns) internal view returns (bytes memory) {
```

Comment:
I didn't updating this function to the new format because we don't check the contents of the batch calldata and the format is still subject to change.

If it makes sense to update this to the current version 0 format, I can add an RLPWriter library and encode correctly.